site/homepage/content/release-notes.md

```diff
@@ -16,6 +16,8 @@ title = "Release Notes"
   configured using the `${PREFIX}_RECONNECT_*` configuration variables. Please
   refer to the [Hono Client Configuration guide](https://www.eclipse.org/hono/docs/latest/admin-guide/hono-client-configuration/)
   for details regarding these new variables.
+* The *message limit* checks is now extended to include command and control messages. 
+  Please refer to the [resource limits](https://www.eclipse.org/hono/docs/latest/concepts/resource-limits/) for details.
```

Comment:
please use the `%doclink` shortcode instead of an explicit URL

Reply:
I have updated and pushed the changes.